src/components/StepBar.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { Step } from 'semantic-ui-react';
 
-const StepBar = ({ steps, currentStepIndex, goToStep, snippets }) => {
+import { STEPS } from '../constants';
```

Comment:
@todo for a later refactor - the name and location of `constants` is a big ambiguous. We have plenty of them around - a bunch for the benefit program logic, etc. These constants are specifically for form sections. They might find a better home with the form sections themselves and maybe a more specific name until we have something we need to add to the file.

Reply:
Good point. We could have a single master constants file, but you're right that it might be better to have more specific constants modules.